jwt/jwt.go

```diff
@@ -0,0 +1,37 @@
+package jwt
```

Comment:
jwt is related to api, so there is no need to exist another package with such name...